Source/d_main.c

```diff
@@ -598,8 +598,9 @@ void D_AddFile(const char *file)
   static int numwadfiles, numwadfiles_alloc;
 
   if (numwadfiles >= numwadfiles_alloc)
-    wadfiles = realloc(wadfiles, (numwadfiles_alloc = numwadfiles_alloc ?
-                                  numwadfiles_alloc * 2 : 8)*sizeof*wadfiles);
+    wadfiles = Z_Realloc(wadfiles, (numwadfiles_alloc = numwadfiles_alloc ?
+                                  numwadfiles_alloc * 2 : 8)*sizeof*wadfiles,
+                        PU_STATIC, 0);
```

Comment:
Is this properly indented? It looks weird on 
https://github.com/fabiangreffrath/woof/pull/551/files

Reply:
Not sure how to fix it. BTW maybe we should use `I_Realloc` here?